### ISSUE TITLE
rsnapshot creates hardlinked symlinks

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg-import#23]
when borg encounters them, it emits this for each:
```
hardlinked symlinks will be archived as non-hardlinked symlinks!
```

TODO:
- check whether that is harmless
- add some way to suppress all these warnings

**Comment by darkdane** [comment]
We have the same problem here. Since upgrading to Borgbackup 1.1 every backup creates this error message.

**Comment by ThomasWaldmann** [comment]
It's just warnings. The symlinks will be backed up, but they won't be hardlinked together any more.

@darkdane do you back up a rsnapshot or rsync+hardlinks backup with borg or where do you see these warnings? I assume you are not running borg-import all the time, do you?

**Comment by darkdane** [comment]
I have never used borg-import on this server. Also we have no rsnapshort or other rsync+hardlinks backup systems running on this machine.

Is there a way to expand the output of borg to see WHERE these warnings are generated? Aktually i can't see which hardlinks he means by giving me this error... :)

**Comment by ThomasWaldmann** [comment]
Guess if you run borg create with --list, the warning will be close to the filename causing it.

**Comment by darkdane** [comment]
Ah i got it with the --list parameter. It's a software we use on this server which uses hardlinks for updates. Because it keeps multiple software versions it hardlinks them to safe space. I can't change this. Is it possible to implement an option to ignore this warning? 

1.0.10 did backups of this system without a load of warnings... :)

**Comment by ThomasWaldmann** [comment]
The warning was implemented after I refactored some code (for 1.1) and realized that it is not possible to archive hardlinked symlinks with the current archive format. 1.0 does it the same way, just does not emit this warning.

For many applications this is relatively harmless because if you extract it, you will still have all the symlinks, they will just not be hardlinked together.

I'll see what I can do. Guess either warn only once or create some means to suppress it.
